fast_forward/encoder/avg.py

```diff
@@ -67,39 +72,45 @@ def __init__(
         self.index = index
         self.w_method = w_method
         self.k_avg = k_avg
+        self.n_weights = k_avg + 1  # +1 for q_emb
```

Comment:
Only +1 if w_method=LEARNED right?
Test this after changing though by running w_method=SOFTMAX_SCORES and w_method=LEARNED.